main/solution/infrastructure/config/infra/cloudformation.yml

```diff
@@ -11,6 +11,59 @@ Conditions:
 
 Description: (SO0144) Service Workbench on AWS Solution
 
+Mappings:
+  ELBAccountIDMapping:
```

Comment:
Which accounts are these?

Reply:
I think it's from this: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html#access-logging-bucket-permissions

Reply:
Yes to Tim's link!